build-system/tasks/karma.conf.js

```diff
@@ -62,9 +62,7 @@ module.exports = {
     watch: true,
     debug: true,
     basedir: __dirname + '/../../',
-    transform: [
-      ['babelify', {compact: false, sourceMapsAbsolute: true}],
-    ],
+    transform: ['babelify'],
```

Comment:
do we no longer need `sourceMapsAbsolute` i remember i turned it on for better stack traces in unit tests

Reply:
Yes, we do. I'll try to add it to `babel.config.js` as a dynamic config.